c/validator_utils.h

```diff
@@ -91,11 +91,31 @@ typedef struct gw_context_t {
   gw_call_receipt_t receipt;
 } gw_context_t;
 
+int _ensure_account_exists(gw_context_t *ctx, uint32_t account_id) {
+  uint8_t script_hash[32];
```

Comment:
Use `ctx->account_count` to check the validity of an account. the account exists if account_id is less than account_count, otherwise, the account does not exist.